code/extensions/che-github-authentication/extension.webpack.config.js

```diff
@@ -26,10 +27,23 @@ module.exports = withDefaults({
         extension: './src/extension.ts',
     },
     externals: {
-		'bufferutil': 'commonjs bufferutil', // ignored
-		'utf-8-validate': 'commonjs utf-8-validate', // ignored
-	},
-	plugins: [
-		new webpack.ContextReplacementPlugin(/keyv/), // needs to exclude the package to ignore warnings https://github.com/jaredwray/keyv/issues/45
-	],
+        'bufferutil': 'commonjs bufferutil', // ignored
+        'utf-8-validate': 'commonjs utf-8-validate', // ignored
+    },
+    plugins: [
+        new webpack.ContextReplacementPlugin(/keyv/), // needs to exclude the package to ignore warnings https://github.com/jaredwray/keyv/issues/45
+    ],
 });
+
+module.exports = merge(config, {
+    module: {
+        rules: [
+            {
+                test: /\.m?js$/,
+                resolve: {
+                    fullySpecified: false, // This avoids the issue with the devfile/api extension requirement
```

Comment:
Could you add a bit more info?
See https://github.com/che-incubator/che-code/pull/396/files#r1714380676